api/handlers/stripeRouterCheckout.js

```diff
@@ -0,0 +1,76 @@
+
+const YOUR_DOMAIN = 'https://d1ujc5c60bkv1u.cloudfront.net';
+const Pool = require('pg').Pool;
+const pool = new Pool({
+    user: 'masteruser',
+    password: "ecommdbpw",
+    host: 'ecommdb.covnxnivrrc5.ap-southeast-2.rds.amazonaws.com',
+    database: "ecommdb",
+    port: 5432,
+    ssl: true
+})
+
+module.exports.stripeCheckOut = async (event) => {
+    try {
+      
+        const stripe = require('stripe')('sk_test_51Np5AOLxfJykJ5Q6fYJ4Amv6DmOwzyKOQMGotSZ8A1rkNo7lXv9WmfeozbEJkPj2MCvSHu9Mv6wkTAVdvS0MQ94w00tPwJSP5K');
```

Comment:
Stripe keys should be configured in the environment (or other configuration) rather than be checked in with code.